frb_rust/src/ffi/web.rs

```diff
@@ -266,6 +296,22 @@ impl Transfer for ArrayBuffer {
     }
 }
 
+#[cfg(feature = "chrono")]
+impl Transfer for chrono::DateTime<chrono::Utc> {
+    fn deserialize(value: &JsValue) -> Self {
+        let ms = value.as_f64().unwrap() as i64;
+        let s = (ms / 1_000) as i64;
+        let ns = (ms.rem_euclid(1_000) * 1_000_000) as u32;
+        Self::from_utc(chrono::NaiveDateTime::from_timestamp(s, ns), chrono::Utc)
+    }
+    fn serialize(self) -> JsValue {
+        self.timestamp_millis().into()
+    }
+    fn transferables(&self) -> Vec<JsValue> {
+        vec![self.serialize()]
+    }
+}
```

Comment:
@Desdaemon should I remove this one or implement it for all ?
any recommendation / obvious mistake of mine ?

Reply:
As far as I understood transferables are used for streams (at least I can see calls leading to `transfer!` and `spawn!`) so I implemented them, naively, for all supported chrono types.
I'm considering adding a test like e.g. `handle_stream` in _pure dart_ to assert that everything goes fine.
Holding on for your feedback first.

Reply:
Transferables are mostly for types that are not `Send` but can be sent in a way that makes sense on the Web. I'm sure `i64` is `Send` so you don't have to implement `Transfer`.

Reply:
resolved in [9ad6c8c](https://github.com/fzyzcjy/flutter_rust_bridge/pull/694/commits/9ad6c8caa0a57cf9539dbe14e49413f1b4d63c2d)